examples/xdp_drop_count.rs

```diff
@@ -0,0 +1,110 @@
+extern crate bcc;
+
+use bcc::{
+    BPFBuilder, BccError,
+    XDPMode::{XDP_FLAGS_HW_MODE, XDP_FLAGS_SKB_MODE},
+    XDP,
+};
+use byteorder::{LittleEndian, ReadBytesExt};
+use clap::{App, Arg};
+use std::collections::{hash_map::Entry, HashMap};
+use std::io::Cursor;
+use std::sync::{
```

Comment:
Same here, one for `std::sync::atomic::...` and one for `std::sync::Arc` would better match the style in this repo.

Reply:
Done!